win32-process.gemspec

```diff
@@ -9,7 +11,7 @@ Gem::Specification.new do |spec|
   spec.test_files = Dir["test/*.rb"]
   spec.files      = `git ls-files -z`.split("\x0").reject { |f| f.match(/^(\..*|Gemfile|Rakefile|examples|test|CHANGELOG.md|README.md)/) }
 
-  spec.required_ruby_version = ">= 2.4" # required for the use of Integer
+  spec.required_ruby_version = ">= 2.7" # required for the use of Integer
```

Comment:
Does this PR need to be tagged with Major version bump label since we are dropping support for the older versions?

Reply:
I found a discussion around this topic (not chef related) that says a minor version bump is appropriate here.